CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ include(GNUInstallDirs)
 #######################
 
 # Add search directory for CMake on OS X
+list(APPEND CMAKE_PREFIX_PATH "/usr/lib/OGRE/OGRE/cmake")
```

Comment:
I think you need to fix or move the comment above this, unless your fix is for macOS?

Reply:
Exactly, it is for Arch Linux. I did not pay attention to the comment. However, at the end I need to use `uname -a` with `EXECUTE_PROCESS()` to parse the kernel name and be sure that it is  Arch Linux. Because usually` lsb_release -is ` is only for Ubuntu

Reply:
I remember adding something like `PKG_CONFIG=/opt/OGRE-1.9` as an env variable in Arch Linux and that did the job.

Reply:
Closed in favor of https://github.com/PX4/sitl_gazebo/pull/304